src/exchange.rs

```diff
@@ -118,6 +168,12 @@ mod tests {
 
         let fetched_rate = exchange.get_rate(usd, gbp).unwrap();
         assert_eq!(fetched_rate.rate, dec!(1.6));
+
+        let fetched_reverse_rate = exchange.get_rate(eur, usd).unwrap();
+        assert_eq!(fetched_reverse_rate.rate, dec!(1) / dec!(1.5));
+
+        let fetched_reverse_rate = exchange.get_rate(gbp, usd).unwrap();
+        assert_eq!(fetched_reverse_rate.rate, dec!(1) / dec!(1.6));
```

Comment:
i dont know if we handle the case where someone sets the exchange rate to 0 well. it was probably OK when we didn't have an inverse rate but since this will raise new errors (1/0) we should add explicit tests cases for that.